spec/calculator_spec.rb

```diff
@@ -1 +1,89 @@
-# Your code here
\ No newline at end of file
+require 'calculator'
+describe Calculator do
```

Comment:
create a common instance for `Calculator` and use it throughout the specs.
`let(:c) { Calculator.new }`